src/main/java/core/basesyntax/db/StorageDaoImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+
+public class StorageDaoImpl implements StorageDao {
+    @Override
+    public void add(String product, Integer value) {
+        if (value >= 0) {
+            Storage.STOCK.put(product, value);
+            return;
+        }
+        throw new RuntimeException("Amount of products '" + product + "' can`t be less than 0");
+    }

```

Comment:
You can simplify this, just at the beginning check negative value, so you don't need to use return operator.
```suggestion
    public void add(String product, Integer value) {
        if (value < 0) {
        throw new RuntimeException("Amount of products '" + product + "' can`t be less than 0");
        }
        Storage.STOCK.put(product, value);
        
    }

```